docs/ga-requirements.md

```diff
@@ -0,0 +1,51 @@
+P0 (e.g. cannot GA without these):
+* End-to-end tests
+  * OTLP, Jaeger and Zipkin
+  * Tomcat, Wildfly, Spring Boot
+  * All Java LTS versions + latest Java version if non-LTS
+* Benchmarking
+  * Startup overhead benchmark
+  * Runtime overhead benchmark
+* Implement all required semantic attributes
+  * Non-required semantic attributes are nice to have, but not required for GA
+* Basic configuration points
+  * Add custom auto-instrumentation
+  * Ability to build "custom distro"
+* Documentation (tag "documentation" minus tag "contributor experience")
+  * All configuration options
+    * Standard OpenTelemetry SDK + Exporter configuration options
+    * Auto-instrumentation configuration options (e.g. disable/enable, peer.service mapping)
+  * For each instrumentation
+    * Document all spans that it captures
```

Comment:
I don't think I've ever seen such detailed documentation per-instrumentation in other tracing libraries. More docs is always better than less, but I don't think it's P0, maybe P2, and it needs to come with the commitment that we will keep the docs up to date.

Reply:
Unfortunately must agree with Anuraag.